### DISCUSSION TITLE
Kafka UI , zookeeper status showing offline ,

### DISCUSSION THREAD

**Discussion by ramamanoj142000** [provectus/kafka-ui#1628] [Q&A] [answered]
Kafka UI , zookeeper status showing offline , how i can pass zookeeper parameter while running docker image . 

 docker run -p 9095:8080 \
    -e KAFKA_CLUSTERS_0_NAME=LOCAL \
    -e KAFKA_CLUSTERS_0_BOOTSTRAPSERVERS=isds-kafkaservice:9094 \
    -e KAFKA_CLUSTERS_0_PROPERTIES_SECURITY_PROTOCOL=SASL_SSL \
    -e KAFKA_CLUSTERS_0_PROPERTIES_SASL_MECHANISM=SCRAM-SHA-512 \
    -e KAFKA_CLUSTERS_0_PROPERTIES_SASL_JAAS_CONFIG="org.apache.kafka.common.security.scram.ScramLoginModule required username="kafka_admin" password="O9csbZJJtGjURn2Akl";" \
    -d isds-dataanalytics/isds-provectus-kafka-ui:latest

**Comment by Haarolean** [comment, marked as answer]
https://github.com/provectus/kafka-ui/blob/master/documentation/compose/kafka-ui.yaml#L20
 KAFKA_CLUSTERS_0_ZOOKEEPER: zookeeper0:2181

**Reply by ramamanoj142000** [reply]
> https://github.com/provectus/kafka-ui/blob/master/documentation/compose/kafka-ui.yaml#L20 KAFKA_CLUSTERS_0_ZOOKEEPER: zookeeper0:2181

i have tried this  but getting error 

 docker run -p 9095:8080 \
     -e KAFKA_CLUSTERS_0_NAME=isds-kafkaservice  \
     -e KAFKA_CLUSTERS_0_BOOTSTRAPSERVERS=isds-kafkaservice :9094 \
    -e KAFKA_CLUSTERS_0_ZOOKEEPER=isds-kafkaservice-dev-west2-lzk01:2185 \
     -e KAFKA_CLUSTERS_0_PROPERTIES_SECURITY_PROTOCOL=SASL_SSL \
    -e KAFKA_CLUSTERS_0_PROPERTIES_SASL_MECHANISM=SCRAM-SHA-512 \
     -e KAFKA_CLUSTERS_0_PROPERTIES_SASL_JAAS_CONFIG="org.apache.kafka.common.security.scram.ScramLoginModule required username="kafka_admin" password="xxxxx";" \
     -d provectuslabs/kafka-ui:latest

docker: Error response from daemon: mkdir /var/lib/docker/overlay2/d78ee2c4a98f332cac3375890daae10cb44a0f52243b33c5c1b0748be3354d49-init: read-only file system.
See 'docker run --help'.

**Reply by Haarolean** [reply]
That's unrelated at all. Which docker daemon is this, which OS? Which user are you running this command us? Which user docker daemon is being run as?

**Reply by ramamanoj142000** [reply]
Yes , you are right this issue was with docker . Thanks you very much for your suggesion . 

I would like to contribute on this project . how i can contribute in this .

**Reply by Haarolean** [reply]
Please refer to [CONTRIBUTING.md](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md)
